src/assets/js/archive.js

```diff
@@ -5,62 +5,109 @@ const releasesToShow = 99999;
 function fetchFlutterReleases(os, callback, errorCallback) {
   // OS: windows, macos, linux
   const url = `https://storage.googleapis.com/flutter_infra_release/releases/releases_${os}.json`;
-  $.ajax({
-    type: 'GET',
-    url: url,
-    dataType: 'json',
-    success: function (data) { callback(data, os); },
-    error: function (xhr, textStatus, errorThrown) {
-      if (errorCallback) errorCallback(os);
-    }
-  })
+  fetch(url, { type: 'GET'})
+    .then(response => response.json())
+    .then(data => callback(data, os))
+    .catch(_ => { if (errorCallback) errorCallback(os) })
 }
 
 function updateTable(releases, os) {
   const releaseData = releases.releases;
 
   for (const channel in releases.current_release) {
-    const table = $(`#downloads-${os}-${channel}`);
-    table.addClass('collapsed').find('.loading').remove();
+    const table = document.getElementById(`downloads-${os}-${channel}`);
+
+    //table is not present when channel is dev
```

Comment:
Even though this is JS not Dart, we prefer to [format comments like sentences](https://dart.dev/effective-dart/documentation#do-format-comments-like-sentences).

```suggestion
    // Table is not present when the channel is `dev`.
```